.github/workflows/ci.yml

```diff
@@ -30,6 +36,13 @@ jobs:
               libasound2-dev \
               libudev-dev
       - uses: actions/checkout@v3
+      - name: Use WASM target for Clippy
```

Comment:
```suggestion
      - name: Install WASM Target
```

Small nit. I feel like this makes the purpose of the stage more clear.

Reply:
Updated :+1: